src/components/CompareResults/ModeInterpretation.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box } from '@mui/material';
+
+import { MANN_WHITNEY_U } from '../../common/constants';
+import { useAppSelector } from '../../hooks/app';
+import { Colors } from '../../styles/Colors';
+import { MannWhitneyResultsItem } from '../../types/state';
+import { TestVersion } from '../../types/types';
+
+export const ModeInterpretation = ({
+  result,
+  testVersion,
+}: {
+  result: MannWhitneyResultsItem;
+  testVersion: TestVersion;
+}) => {
+  if (!result || !result.silverman_kde || testVersion !== MANN_WHITNEY_U) {
+    return null;
+  }
+
+  function getStyles(theme: string) {
+    const backgroundColor =
+      theme === 'light' ? Colors.Background300 : Colors.Background300Dark;
+
+    return {
+      backgroundColor,
+      display: 'block',
+      alignItems: 'center',
+      margin: '15px',
+      borderRadius: '5px',
+      padding: 2,
+    };
+  }
+  const mode = useAppSelector((state) => state.theme.mode);
+
+  // const result?.silverman_kde?.mode_comments
+  return (
+    <Box sx={getStyles(mode)}>
+      <table>
+        <thead>
+          <tr
```

Comment:
Question: will there be multiple rows here? I noticed in the design we have mode 1 and mode 2. Are there more than 1 modes ? Because it's confusing me here. 

I'm asking because if the modes are in array of some sort or it would be better to map through that array to generate the rows instead of to hard code it like here. Please let me know if I should clarify. I could also be misunderstanding how the data is stored.

Reply:
There's potentially more than 1 mode. I need to update it on the backend to become a returning list. The shape of the data coming back is not quite right.